pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -40,22 +41,22 @@
 
   BOOLEAN {
     @Override
-    public Integer toInteger(Object value) {
+    public int toInt(Object value) {
       return ((Boolean) value) ? 1 : 0;
     }
 
     @Override
-    public Long toLong(Object value) {
+    public long toLong(Object value) {
       return ((Boolean) value) ? 1L : 0L;
```

Comment:
seems like this should be `1l:0l` instead of `1L : 0L` to avoid unnecessary type conversion?

Reply:
Both `1L` and `1l` represents the primitive long. We usually use capital `L` for long to differentiate it from `1`. Check the first answer here: https://stackoverflow.com/questions/769963/javas-l-number-long-specification